src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -233,6 +234,11 @@ public OpenIdConnectProtocolValidator ProtocolValidator
         public bool RefreshOnIssuerKeyNotFound { get; set; }
 
         /// <summary>
+        /// Determines whether authorization requests should be made using POST.
+        /// </summary>
+        public bool UsePostAuthorizationRequests { get; set; }
```

Comment:
Well, the `OpenID Connect Session Management` specs are very unclear about the methods an authorization server should or must support, since they only use the `redirect` keyword:

> In this case, the RP, after having logged the End-User out of the RP, redirects the End-User's User Agent to the OP's logout endpoint URL.

Personally, I decided to support both methods in `AspNet.Security.OpenIdConnect.Server`, as I couldn't think of a valid reason to reject POST requests.